tests/ReflectionClassTest.php

```diff
@@ -290,4 +300,58 @@ protected function getGettersToCheck()
 
         return $allNameGetters;
     }
+
+    /**
+     * Returns list of ReflectionProperty that be checked
+     *
+     * @return array
+     */
+    protected function getPropertiesToCheck()
+    {
+        $names = [
+            'name',
+        ];
+
+        return $names;
+    }
+
+    /**
+     * @param string[] $membersToCheck
+     * @return array
+     */
+    protected function caseProvider($membersToCheck)
+    {
+        $testCases = [];
+        $files = $this->getFilesToAnalyze();
+        foreach ($files as $fileList) {
+            foreach ($fileList as $fileName) {
+                $fileName = stream_resolve_include_path($fileName);
+                $fileNode = ReflectionEngine::parseFile($fileName);
+
+                $reflectionFile = new ReflectionFile($fileName, $fileNode);
+                include_once $fileName;
+                foreach ($reflectionFile->getFileNamespaces() as $fileNamespace) {
+                    foreach ($fileNamespace->getClasses() as $refClass) {
+                        $qcn = ltrim($refClass->getName(), '\\'); // workaround for #80
```

Comment:
This PR should be ltrimming class name. Why we need to also ltrim in tests?

Reply:
Tests should check that no difference with and without leading backslash between native Reflection* and Go\ParserReflection\Reflection*. It is common consists checking regardless fix in the PR.

Reply:
I'm still not getting it. If this PR normalizes stuff so that there won't be any leading `\` in class names around the library and built-in reflection already does it, then why we need to ltrim it in tests?